sources/packages/backend/apps/api/src/route-controllers/application-offering-change-request/application-offering-change-request.institutions.controller.ts

```diff
@@ -271,6 +274,7 @@ export class ApplicationOfferingChangeRequestInstitutionsController extends Base
   @ApiUnauthorizedResponse({
     description: "The location does not have access to the offering.",
   })
+  @HasLocationAccess("locationId", [InstitutionUserTypes.user])
```

Comment:
The decorator `HasLocationAccess` is relying on the `InstitutionUserTypes` from `../../auth` and we should have it also imported from the same, as blow.
```ts
import { InstitutionUserTypes } from "../../auth";
```
Please review the other controllers that are importing the same.

Reply:
I can see some controllers still importing from `@sims/sims-db`. Were they changed?
- application-offering-change-request.institutions.controller.ts
- confirmation-of-enrollment.institutions.controller.ts
- education-program-offering.institutions.controller.ts
- program-info-request.institutions.controller.ts

Reply:
I checked all the controllers now. Thanks!